pyiron/lammps/control.py

```diff
@@ -181,8 +181,8 @@ def load_default(self, file_content=None):
                 + "include             potential.inp\n"
                 + "fix___ensemble      all nve\n"
                 + "variable___dumptime equal 100\n"
-                + "dump___1            all custom ${dumptime} dump.out id type xsu ysu zsu fx fy fz\n"
-                + 'dump_modify___1     sort id format line "%d %d %20.15g %20.15g %20.15g %20.15g %20.15g %20.15g"\n'
+                + "dump___1            all custom ${dumptime} dump.out id type xsu ysu zsu fx fy fz vx vy vz\n"
+                + 'dump_modify___1     sort id format line "%d %d %20.15g %20.15g %20.15g %20.15g %20.15g %20.15g %20.15g %20.15g %20.15g"\n'
```

Comment:
This isn't anything to do with your changes, but let's use the opportunity to make line 185 consistently use double quotations